parm/config/config.vrfy

```diff
@@ -142,22 +141,25 @@ fi
 # Cyclone genesis and cyclone track verification
 #-------------------------------------------------
 
-export ens_tracker_ver=v1.1.15.1
+export ens_tracker_ver=v1.1.15.5
 if [ $machine = "WCOSS_DELL_P3" ] ; then
     export ens_tracker_ver=v1.1.15.3
 fi
 export HOMEens_tracker=$BASE_GIT/tracker/ens_tracker.${ens_tracker_ver}
 if [ $machine = "ORION" ] ; then
     export HOMEens_tracker=$BASE_GIT/tracker/TC_tracker.v1.1.15.2
 fi
-
+if [ $machine = "WCOSS2" ] ; then
+    export HOMEens_tracker=/lfs/h2/emc/global/noscrub/emc.global/git/ens_tracker.${ens_tracker_ver}
```

Comment:
I believe @KateFriedman-NOAA will soon have an installation of the tracker in `BASE_GIT`.  Is there a reason to not have `$BASE_GIT` instead of the hard-wired path?

Reply:
Similar to the comment above. This is temporary.